hack/build-cross.sh

```diff
@@ -33,6 +33,12 @@ os::build::build_binaries "${OS_CROSS_COMPILE_TARGETS[@]}"
 OS_BUILD_PLATFORMS=("${OS_IMAGE_COMPILE_PLATFORMS[@]-}")
 os::build::build_static_binaries "${OS_IMAGE_COMPILE_TARGETS[@]-}" "${OS_SCRATCH_IMAGE_COMPILE_TARGETS[@]-}"
 
+# Build dockerregistry
+(
```

Comment:
Both - the vendor pattern and build changes.  I want to establish a pattern
we can use if this comes up in other paths.

On Jun 9, 2016, at 10:28 AM, Alexey Gladkov <notifications@github.com>
wrote:

In hack/build-cross.sh
<https://github.com/openshift/origin/pull/8938#discussion_r66451285>:

> @@ -33,6 +33,12 @@ os::build::build_binaries "${OS_CROSS_COMPILE_TARGETS[@]}"
>  OS_BUILD_PLATFORMS=("${OS_IMAGE_COMPILE_PLATFORMS[@]-}")
>  os::build::build_static_binaries "${OS_IMAGE_COMPILE_TARGETS[@]-}" "${OS_SCRATCH_IMAGE_COMPILE_TARGETS[@]-}"
>
> +# Build dockerregistry
> +(

@smarterclayton <https://github.com/smarterclayton> You mean changes in
hack/build-cross.sh or directory in general?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/openshift/origin/pull/8938/files/bba3a7d7bd09c671c94531a774860945ca9e5953..4390daddfbe7ffb13344f4ef78362ff54aeb046a#r66451285>,
or mute the thread
<https://github.com/notifications/unsubscribe/ABG_px_6WqZs-tyszVmdoQTSYW6Psi9Yks5qKCMagaJpZM4IiLYZ>
.